mandatory/2-function-creation.js

```diff
@@ -25,7 +43,11 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {   // comment: i don't understand
+
```

Comment:
I think this function is an opportunity to practice some of the methods you have learned up to now. Let's break down the requirements:

- takes an array of strings as input => the parameter will take an array and we want to do the same operation or set of operations on each string in the array

What operations should we do?
- removes any spaces in the beginning or end of each string => is there a useful way to `trim` these strings?
- removes any forward slashes (/) in each string => Can we `replace` the forward slashes? Look in the mdn docs
- makes all strings all lowercase => what about here? Can you think of a method that could achieve this?

Once we have sorted out what we will do to each string, we need to find a way to apply this to each string programmatically. Is there a way to loop over every item in your array and apply these transformations? Look through the syllabus for this week to help you.